commands/operator-sdk/cmd/cmdutil/util.go

```diff
@@ -74,3 +78,15 @@ func GetCurrPkg() string {
 	currPkg := strings.Replace(wd, goSrc+string(filepath.Separator), "", 1)
 	return currPkg
 }
+
+// GetOperatorType returns type of operator is in cwd
+// This function should be called after verifying the user is in project root
+// e.g: "go", "ansible"
+func GetOperatorType() string {
+	// Assuming that if config/config.yaml exists then this is a Go operator
+	_, err := os.Stat(configYaml)
```

Comment:
We're actually removing config.yaml in the refactor branch.

So we can change this later on as well but might be better to check for something else.
How about `Gopkg.toml`? That should always be present at the root of a Go type project.

Reply:
Much smarter, thank you :+1: